src/libraries/System.IO.FileSystem.AccessControl/ref/System.IO.FileSystem.AccessControl.cs

```diff
@@ -118,4 +118,21 @@ public abstract partial class FileSystemSecurity : System.Security.AccessControl
         public void SetAccessRule(System.Security.AccessControl.FileSystemAccessRule rule) { }
         public void SetAuditRule(System.Security.AccessControl.FileSystemAuditRule rule) { }
     }
+    public class MemoryMappedFileSecurity : ObjectSecurity<System.IO.MemoryMappedFiles.MemoryMappedFileRights>
```

Comment:
I noticed I placed this class in the wrong namespace. I have to move it to System.IO.MemoryMappedFiles, like we have it in [netfx](https://apisof.net/catalog/System.IO.MemoryMappedFiles.MemoryMappedFileSecurity).